__tests__/integration/handlers/application.js

```diff
@@ -0,0 +1,189 @@
+const applicationHandler = require('../../../src/handlers/applications')
+const { Applications } = require('../../../src/database')
+
+const APP_ID = 'create_handler_1'
+
+describe('Application handler', () => {
+  beforeEach(() => {
+    return Applications.delete(APP_ID).catch()
+  })
+
+  describe('create',() => {
+    it('creating using only app_id', async () => {
+      const {
+        body,
+        statusCode,
+      } = await applicationHandler.create({
+        body: `{"app_id": "${APP_ID}"}`,
+      })
+
+      const response = JSON.parse(body)
+
+      expect(statusCode).toBe(201)
+      expect(response.code).toBe('created')
+      expect(response.data).toHaveProperty('app_id', APP_ID)
+      expect(response.data).toHaveProperty('created_at')
+      expect(response.data).toHaveProperty('updated_at')
+      expect(response.data).toHaveProperty('status', 'active')
+      expect(response.data).toHaveProperty('api_key')
+    })
+
+    it('creating without app_id', async () => {
+      const {
+        body,
+        statusCode,
+      } = await applicationHandler.create({
+        body: `{}`,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
        body: "{}",
```